src/kubernetes-tool/templates/categorisation_view.vue

```diff
@@ -1,41 +1,99 @@
+<!--
+Copyright 2019-2020 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
 <template>
     <div>
         <hr :style="{marginTop: '10px'}">
-        <p>When the chart is deployed, it should cost you an extra ${{ cost }} in additional resources such as block storage or load balancers.</p>
+        <p>When the chart is deployed, it should cost you an extra ${{ Categorisation.getCost() }} in additional resources such as block storage or load balancers.</p>
         <hr :style="{marginTop: '10px'}">
-        <div v-for="(key, index) in all.keys()">
-            <h1>{{ key.name }}</h1>
-            <p v-if="key.description">
-                {{ key.description }}
-            </p>
-            <p>
-                <a v-for="item in all.get(key)" :key="item.fp" class="button is-small" :style="{marginRight: '10px', marginBottom: '10px'}" @click="emitFp(item.fp)">
-                    {{ item.fp }}
-                </a>
-            </p>
-            <hr v-if="index !== Array.from(all.keys()).length - 1">
+        <div v-for="allCats in [Categorisation.getAll()]">
+            <div v-for="(key, index) in allCats.keys()" :key="key.name">
+                <h2>{{ key.name }}</h2>
+                <p v-if="key.description">
+                    {{ key.description }} <a v-if="key.learnMore" :href="key.learnMore">Learn more...</a>
```

Comment:
Use the do-vue ExternalLink component for the learn more link, as they're external.